odata-url-conventions/5 Query Options.md

```diff
@@ -2190,6 +2144,56 @@ http://host/service/Products?$expand=$value
 ```
 :::
 
+#### ##subsubsubsec Expand Options
+
+Query options can be applied to an expanded navigation property by
+appending a semicolon-separated list of query options, enclosed in
+parentheses, to the navigation property name.
+The system query option, irrespective of casing or whether or not it is prefixed with a `$`,
+MUST NOT be specified more than once in the list.
+Allowed system query options are
+[`$compute`](#SystemQueryOptioncompute),
+[`$select`](#SystemQueryOptionselect),
+`$expand`, and
+[`$levels`](#ExpandOptionlevels) for all navigation properties, plus
+[`$filter`](#SystemQueryOptionfilter),
+[`$orderby`](#SystemQueryOptionorderby),
+[`$skip`](#SystemQueryOptionstopandskip), [`$top`](#SystemQueryOptionstopandskip),
+[`$count`](#SystemQueryOptioncount), and
+[`$search`](#SystemQueryOptionsearch)
+for collection-valued navigation properties.
```

Comment:
Why is this an explicit list whereas section "Select Options" below refers to section "System Query Options"?

Reply:
This was existing wording for both $select and $expand that I copied into new respective Select Options/Expand Options for each.

I've made them consistent.